src/main/java/io/stargate/sgv2/jsonapi/api/model/command/table/definition/datatype/VectorColumnDesc.java

```diff
@@ -6,17 +6,18 @@
 import io.stargate.sgv2.jsonapi.service.schema.tables.ApiVectorType;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
 
 /** Column type for {@link ApiVectorType} */
 public class VectorColumnDesc extends ComplexColumnDesc {
   public static final FromJsonFactory FROM_JSON_FACTORY = new FromJsonFactory();
 
   // Float will be default type for vector
   private final ColumnDesc valueType;
-  private final int dimensions;
+  private final Integer dimensions;
```

Comment:
I guess this is existing field, but Vector has just one `dimension`, not multiple `dimensions`?